CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Airbrake Changelog
 
 ### master
 
+* **IMPORTANT:** added Shoryuken integration
+  ([#669](https://github.com/airbrake/airbrake/pull/669))
```

Comment:
I'll release it this week. I plan to release airbrake-ruby v1.8.0 and this repo will need some adjustments.